content/articles/heroku-service.md

```diff
@@ -23,19 +23,24 @@ This service is being deprecated. We recommend that you use the [Heroku Connecto
 
 ## Enabling Heroku service
```

Comment:
not doing extensive edits on this one since it's deprecated — just making it match the others and updating the screenshot.